packages/gridstack/src/components/SaveDashboardModal/SaveDashboardModal.vue

```diff
@@ -11,6 +11,11 @@
       name="name"
       class="base-input"
     />
+    <div v-if="showError" class="error-message">
+      Fehler beim Speichern der Konfiguration. Konfiguration mit diesem Namen
+      existiert bereits.
+    </div>
```

Comment:
If this should look like an error box we have a component for that. See https://lion5.github.io/component-library/?path=/docs/core_components-boxes-errorbox--documentation

Reply:
Changed it to use ErrorBox